web-console/project-details/in-project-catalog.md

```diff
@@ -0,0 +1,11 @@
+
+# In-project Catalog
+![Catalog inside of a project](img/in-project-catalog.png)
+- Users with permission to add to a project will see a `Catalog` option in the main navigation.
+  - The Catalog menu should use `fa-book`.
```

Comment:
Can you check with @jennyhaines - I thought they were supposed to use pficon-service-catalog.  There's a follow on story to update that PF icon so that both OpenShift & ManageIQ use the same one.

Reply:
I'm not positive, but I thought we decided to use fa-book until the pficon-service-catalog icon was updated.

Reply:
I had that originally and Jeff indicated that we were using `fa-book` at least temporarily. If the plan is to eventually use an updated `pficon-service-catalog` I can change to docs to say that and it will eventually be true. Or I can leave it as is and update the icon and the screenshot at the same time once we have the new icon.

Reply:
Yes, @serenamarie125 - I had said that I was okay with using "fa-book" until "pficon-service-catalog" was updated.

Reply:
But that means that the OpenShift code will need to be modified to get the new icon. If they use pficon-service-catalog, then it will be updated automatically in OpenShift when PF updates the icon. @jennyhaines @jeff-phillips-18 @ncameronbritt